### PR TITLE
[DOC] Updates to the upgrade chapter [ENTMQST-781]

### PR DESCRIPTION
**Pull request by PaulRMellor**
Documentation

Updates for the upgrade chapter

### CODE REVIEW

File: books/proc-updating-zookeeper-binaries.adoc [hunk 1]
```diff
@@ -6,20 +6,45 @@
...
+----
 
-. Restart the Zookeeper server.
+. Copy the `libs` `bin` and `docs` directories from the temporary directory:
 +
 [source,shell,subs=+quotes]
 ----
-su - kafka
-/opt/kafka/bin/zookeeper-server-start.sh -daemon /opt/kafka/config/zookeeper.properties
+cp -r /tmp/kafka-1-1-0/libs /opt/kafka/
```
Comment (scholzj) [inline books/proc-updating-zookeeper-binaries.adoc]:
```suggestion
cp -r /tmp/kafka-1-1-0/kafka_y.y-x.x.x/libs /opt/kafka/
```

File: books/proc-updating-zookeeper-binaries.adoc [hunk 2]
```diff
@@ -6,20 +6,45 @@
...
+cp -r /tmp/kafka-1-1-0/bin /opt/kafka/
```
Comment (scholzj) [inline books/proc-updating-zookeeper-binaries.adoc]:
```suggestion
cp -r /tmp/kafka-1-1-0/kafka_y.y-x.x.x/bin /opt/kafka/
```

File: books/proc-updating-zookeeper-binaries.adoc [hunk 3]
```diff
@@ -6,20 +6,45 @@
...
+cp -r /tmp/kafka-1-1-0/docs /opt/kafka/
```
Comment (scholzj) [inline books/proc-updating-zookeeper-binaries.adoc]:
```suggestion
cp -r /tmp/kafka-1-1-0/kafka_y.y-x.x.x/docs /opt/kafka/
```

File: books/proc-upgrading-kafka-brokers-to-amq-streams-1-1-0.adoc [hunk 4]
```diff
@@ -30,11 +30,20 @@ mkdir /tmp/kafka-1-1-0
...
 ----
-sudo mv /tmp/kafka-1-1-0/kafka_y.y-x.x.x/* /opt/kafka/
+rm -rf /opt/kafka/libs /opt/kafka/bin /opt/kafka/docs
+----
+
+. Copy the `libs` `bin` and `docs` directories from the temporary directory:
++
+[source,shell,subs=+quotes]
+----
+cp -r /tmp/kafka-1-1-0/libs /opt/kafka/
```
Comment (scholzj) [inline books/proc-upgrading-kafka-brokers-to-amq-streams-1-1-0.adoc]:
```suggestion
cp -r /tmp/kafka-1-1-0/kafka_y.y-x.x.x/libs /opt/kafka/
```

File: books/proc-upgrading-kafka-brokers-to-amq-streams-1-1-0.adoc [hunk 5]
```diff
@@ -30,11 +30,20 @@ mkdir /tmp/kafka-1-1-0
...
+cp -r /tmp/kafka-1-1-0/bin /opt/kafka/
```
Comment (scholzj) [inline books/proc-upgrading-kafka-brokers-to-amq-streams-1-1-0.adoc]:
```suggestion
cp -r /tmp/kafka-1-1-0/kafka_y.y-x.x.x/bin /opt/kafka/
```

File: books/proc-upgrading-kafka-brokers-to-amq-streams-1-1-0.adoc [hunk 6]
```diff
@@ -30,11 +30,20 @@ mkdir /tmp/kafka-1-1-0
...
+cp -r /tmp/kafka-1-1-0/docs /opt/kafka/
```
Comment (scholzj) [inline books/proc-upgrading-kafka-brokers-to-amq-streams-1-1-0.adoc]:
```suggestion
cp -r /tmp/kafka-1-1-0/kafka_y.y-x.x.x/docs /opt/kafka/
```

File: books/proc-upgrading-kafka-brokers-to-amq-streams-1-1-0.adoc [hunk 7]
```diff
@@ -6,7 +6,7 @@
 
 = Upgrading Kafka brokers to {ProductName} 1.1.0
 
-To upgrade to {ProductName} 1.1.0 on the host machine, download and extract the *Red Hat AMQ Streams 1.1.0* archive from the Red Hat Customer Portal. 
+To upgrade to {ProductName} 1.1.0 on the host machine, download and extract the *Red Hat AMQ Streams 1.1.0* archive from the Red Hat Customer Portal.
```
Comment (scholzj) [inline books/proc-upgrading-kafka-brokers-to-amq-streams-1-1-0.adoc]:
```suggestion
To upgrade to {ProductName} 1.1.0 on the host machine, download and extract the *{ProductName} 1.1.0* archive from the Red Hat Customer Portal.
```

File: books/proc-updating-kafka-brokers-to-new-inter-broker-protocol-version.adoc [hunk 8]
```diff
@@ -21,6 +19,8 @@ NOTE: Update and restart the Kafka brokers one-by-one. Before you restart a modi
 
 .Procedure
 
+For all Kafka brokers in your {ProductName} cluster:
```
Comment (tombentley) [inline books/proc-updating-kafka-brokers-to-new-inter-broker-protocol-version.adoc]: It's important this is done one at a time.

```suggestion
For each Kafka broker in your {ProductName} cluster and one at a time:
```

File: books/proc-updating-kafka-brokers-to-new-message-format-version.adoc [hunk 9]
```diff
@@ -23,6 +23,8 @@ NOTE: Update and restart the Kafka brokers one-by-one. Before you restart a modi
 
 .Procedure
 
+For all Kafka brokers in your {ProductName} cluster:
```
Comment (tombentley) [inline books/proc-updating-kafka-brokers-to-new-message-format-version.adoc]:
```suggestion
For each Kafka broker in your {ProductName} cluster and one at a time:
```

File: books/proc-updating-zookeeper-binaries.adoc [hunk 10]
```diff
@@ -6,20 +6,45 @@
 
 = Updating the Zookeeper binaries
 
-Before upgrading to {ProductName} 1.1.0, update your Zookeeper cluster with the latest binaries.
+For all Kafka brokers in your {ProductName} cluster:
```
Comment (tombentley) [inline books/proc-updating-zookeeper-binaries.adoc]:
```suggestion
For each Kafka broker in your {ProductName} cluster and one at a time:
```

File: books/proc-updating-zookeeper-binaries.adoc [hunk 11]
```diff
@@ -6,20 +6,45 @@
...
+cp -r /tmp/kafka-1-1-0/kafka_y.y-x.x.x/docs /opt/kafka/
 ----
 
-. Repeat this procedure on all the nodes of the ZooKeeper cluster.
\ No newline at end of file
+. Delete the temporary directory:
++
+[source,shell,subs=+quotes]
+----
+rrm -rf /tmp/kafka-1-1-0
```
Comment (tombentley) [inline books/proc-updating-zookeeper-binaries.adoc]:
```suggestion
rm -rf /tmp/kafka-1-1-0
```

File: books/proc-updating-zookeeper-binaries.adoc [hunk 12]
```diff
@@ -6,20 +6,45 @@
...
+
+. Restart the Kafka brokers:
```
Comment (tombentley) [inline books/proc-updating-zookeeper-binaries.adoc]:
```suggestion
. Restart the Kafka broker:
```

File: books/proc-upgrading-kafka-brokers-to-amq-streams-1-1-0.adoc [hunk 13]
```diff
@@ -16,11 +16,11 @@ NOTE: This procedure is part of the xref:assembly-upgrade-1-1-0-{context}[Upgrad
 
 .Procedure
 
-Perform the following steps for all Kafka brokers in your {ProductName} cluster:
+For all Kafka brokers in your {ProductName} cluster:
```
Comment (tombentley) [inline books/proc-upgrading-kafka-brokers-to-amq-streams-1-1-0.adoc]:
```suggestion
For each Kafka broker in your {ProductName} cluster and one at a time:
```

**Pull request merged**